ccf/app/views.py

```diff
@@ -0,0 +1,6 @@
+from django.shortcuts import redirect, render
```

Comment:
rendered could be removed

Reply:
Good catch, will remove

Reply:
Addressed in https://github.com/Ross-Alexandra/SENG-371-Project-2/pull/43/commits/a61859e168304df545112c4360af8cfe1fac8f06